draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
+                                        id BIGSERIAL NOT NULL,
+                                        condition TEXT NOT NULL,
+                                        flow_type TEXT NOT NULL,
+                                        event_type TEXT NOT NULL,
+                                        threshold_value FLOAT NOT NULL,
+                                        target_uri TEXT NOT NULL,
```

Comment:
```suggestion
                                        webhook_target_uri TEXT NOT NULL,
```
want to make naming specific as there could be other targets in future